### ISSUE TITLE
ENOENT, missing source/_draft directory

### ISSUE THREAD

**Issue by DavidNeko** [hexojs/hexo#3232]
I was following a basic Hexo tutorial and got stuck when I tried to publish a new post.
I'm sure all dependencies and required stuffs were installed before I do init.
When I tried to publish, it says

`Error: ENOENT: no such file or directory, scandir '/Users/Dav/File/tets/source/_drafts'
`

It seems like the _draft folder is missing but I don't know how to fix it.
I'm using zsh and idk if this is causing any problems. 
Any1 has any clue how to fix this problem?

**Comment by demurgos** [comment]
You could try to create the directory manually, there is no special content in this directory. The error itself is probably a bug: if the directory is missing, it may be better to treat it as empty (and eventually emit a non-fatal warning).

**Comment by DavidNeko** [comment]
Hi @demurgos , thanks for the reply! I got the Hexo server started on my localhost and also tried to make a new draft using
`hexo new draft "My First Blog Post"
gladly it works fine and I could check out that template draft from my localhost after I started server.
I guess the problem that I encountered was that I was trying to publish a post before I create a draft for it. My problem get solved after I made a new draft with the same name as the post before publishing it. 
`hexo new draft $my-draft-name 
solves my problem!
Thanks again!
